sample.env

```diff
@@ -0,0 +1,7 @@
+APP_HOST="0.0.0.0"
+APP_PORT=2113
+OPENAI_URI="https://localhost:11434/v1"
+OPENAI_MODEL="mistral"
+OPENAI_IGNORE_SSL="True"
+MLFLOW_TRACKING_URI="http://localhost:5000"
+MLFLOW_EXPERIMENT_NAME="Tool Based Agent"
```

Comment:
Since you have named this agent as "react agent". You can choose to rename this as well. (Not mandatory though)